### PR TITLE
Added new Terraform tutorial to resources

### PR DESCRIPTION
**Pull request by mariuszmichalowski**

### CODE REVIEW

**Comment by mariuszmichalowski** [comment]
Hey, @lsorber can you take a look?

File: README.md [hunk 1]
```diff
@@ -267,6 +267,7 @@ For more awesomeness, check out [![Awesome](https://cdn.rawgit.com/sindresorhus/
 - :snake: [Terraform best practices](https://github.com/ozbillwang/terraform-best-practices) - Terraform best practices (1 hour)
 - :unicorn: [Terraform pre-commit hooks collection](https://github.com/antonbabenko/pre-commit-terraform) - How to automate Terraform code quality checks with pre-commit (1 hour)
 - :unicorn: [Awesome Terraform](https://github.com/shuaibiyy/awesome-terraform) - An awesome list of Terraform resources (30 min)
+- :hatched_chick: [Terraform Tutorial](https://spacelift.io/blog/terraform-tutorial) - Getting Started With Terraform on AWS (1 hour)
```
Comment (lsorber) [inline README.md:270]:
```suggestion
- :hatched_chick: [Terraform Tutorial](https://spacelift.io/blog/terraform-tutorial) - How to get started with Terraform (1 hour)
```

**Review by lsorber — CHANGES_REQUESTED** [review]
Hi @mariuszmichalowski, thanks for the PR! My apologies for the late response, lost track of this with the summer holidays. I have just one suggestion (so that the description completes the sentence "After reading this article you will have learned ..."), but apart from that this LGTM!

File: README.md (hunk 1 above)
Comment (mariuszmichalowski) [reply README.md]: Hi @lsorber, 
Thanks for getting back to me; no worries at all! I just committed your suggestion; thanks :)

**Pull request merged**